python/cudf/cudf/core/series.py

```diff
@@ -5009,12 +5009,73 @@ def _get_dt_field(self, field):
         )
 
     def ceil(self, field):
+        """
+        Perform ceil operation on the data to the specified freq.
+
+        Parameters
+        ----------
+        field : str
```

Comment:
If this is supposed to match `Series.dt.ceil` from pandas, this parameter should be named `freq` instead of `field`. https://pandas.pydata.org/docs/reference/api/pandas.Series.dt.ceil.html#pandas.Series.dt.ceil